site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -68,6 +68,9 @@ public function newPage($template_id = null) {
         $this->core->getOutput()->addVendorJs(FileUtils::joinPaths('flatpickr', 'plugins', 'shortcutButtons', 'shortcut-buttons-flatpickr.min.js'));
         $this->core->getOutput()->addVendorCss(FileUtils::joinPaths('flatpickr', 'flatpickr.min.css'));
         $this->core->getOutput()->addVendorCss(FileUtils::joinPaths('flatpickr', 'plugins', 'shortcutButtons', 'themes', 'light.min.css'));
+        $this->core->getOutput()->addVendorJs(FileUtils::joinPaths('select2', 'js', 'select2.full.min.js'));
```

Comment:
Just to check - is there a specific reason that the "full" version of select2 is being used over the "standard" version here? (see https://select2.org/getting-started/builds-and-modules for differences)

Reply:
I think you are correct -- just tested, and it is safe to use the standard version here.  Thanks for your review